backend/src/db.js

```diff
@@ -20,38 +20,58 @@ export class MemoryDataSource extends DataSource {
   constructor() {
     super();
 
-    this.usersData = new Set();
+    this.usersData = [];
```

Comment:
warum soll hier nicht ein Set sein?

Reply:
Ich wollte es konsistent zu `postsData` machen (beide liste oder beide set). Und wir benutzen das Set sowieso nicht, man muss in JS equality überschreiben, damit es mit JSON Objekte klappt. Hier ein kurzes Beispiel: https://jsfiddle.net/7gzojkec/